src/data/media-service.ts

```diff
@@ -2,7 +2,7 @@ import { decodeMediaData } from '~/utils/decode-media-data'
 import type { ApiQueryParams } from '~/utils/search-query-transform'
 import type { ApiService } from '~/data/api-service'
 import type { DetailFromMediaType, Media } from '~/models/media'
-import type { SupportedMediaType } from '~/constants/media'
+import { AUDIO, SupportedMediaType } from '~/constants/media'
```

Comment:
```suggestion
import { AUDIO, type SupportedMediaType } from '~/constants/media'
```

Reply:
I didn't know you can add `type` in front of the named import! Thank you, @dhruvkb!